File: pset6/mario/less/mario.py
from cs50 import get_int

while True:
    print("Specify Pyramid Height: ", end = "")

    pyramidheight = get_int()

    if pyramidheight > 0 and pyramidheight <=23:
            break

    else:
            print("Provide a Number between 0 and 23")

for i in range (pyramidheight):
    for j in range (pyramidheight - i - 1):
        print(" ", end = "")
    for k in range (i + 1):
        print("#", end = "")
    print("")

File: pset1/mario/mario.c

#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int height;

//promt height  for pyramid from user
    do
    {
        printf("Specify Pryamid Height:");
        height = get_int();
    }

    while (height < 0 || height > 23);

//for loop to build the pyramid

    for (int row = 0; row < height; row++)
    {
        for (int spaces = height - row; spaces > 1; spaces--)
        {
            printf(" ");
        }
        for (int hashes = 0; hashes < row + 2; hashes++)
        {
            printf("#");
        }

        printf("\n");

    }
}